jenkins/ci.suse.de/ardana-trigger-trackupstream.yaml

```diff
@@ -0,0 +1,81 @@
+- job:
+    name: 'ardana-trigger-trackupstream'
+
+    triggers:
+      - gerrit:
+          server-name: 'gerrit.suse.provo.cloud'
+          trigger-on:
+            - change-merged-event
+            - ref-updated-event
+          skip-vote:
+              notbuilt: true
+          projects:
+            - project-compare-type: 'REG_EXP'
+              project-pattern: "\
+                ardana/(\
```

Comment:
Can we move this pattern to a extra file? I guess we will use it more often.

Reply:
Yaml Anchors & Aliases don't work as is here because they don't carry from templates to the non-template document tree. `!include` from testing doesn't work here, probably as it is only part of a set instead of the whole. I intent to test using `!include-raw` for only the regexp string after this is merged.

Reply:
Proposed in: https://github.com/SUSE-Cloud/automation/pull/2791